README.md

```diff
@@ -65,7 +74,7 @@ Convert the following create method in Mongoose to ActiveRecord.
 How does module.exports help us with separation of concerns?
 
 ```text
-
+It sets up each module as a separate entity and you can reference them throughout your application.
 ```
```

Comment:
👍  anything included in module.exports is available globally to be imported into other files